config/settings.yml

```diff
@@ -103,6 +103,7 @@ agfs_fee_reform_release_date: <%= Date.new(2018, 4, 1) %>
 agfs_scheme_11_release_date: <%= Date.new(2018, 12, 31) %>
 clar_release_date: <%= Date.new(2020, 9, 17) %>
 lgfs_scheme_10_clair_release_date: <%= Date.new(2022, 9, 30) %>
+agfs_scheme_13_clair_release_date: 2022-09-30
```

Comment:
Would it be ideal to note the reason behind the different date format?

Reply:
The plan is to change all the dates to the string format Joe has used in this commit. We just don't want to do it as part of this change, to reduce the risk of changing too many things at once and breaking something. I guess we could add a comment to that effect, but it would (hopefully) be short-lived.

Reply:
That is correct. I intend to change all the dates but as a separate PR. It is possible that it could have some side effects on some tests, but (I'm fairly sure) not on the production code.